### DISCUSSION TITLE
Use drawer start and drawer end on the same page

### DISCUSSION THREAD

**Discussion by mooorphy** [saadeghi/daisyui#1306] [Q&A] [answered]
Hey,

I am currently using daisyUi with Nuxt 3 and when I switch from the drawer end to the drawer start, the drawer flies over the screen once. Is it possible to use the drawer end and drawer start on the same page without this behave?

thanks
Dennis

**Comment by saadeghi** [comment, marked as answer]
See this discussion: https://github.com/saadeghi/daisyui/discussions/674
